### PR TITLE
Split vdom dependency

### PR DESCRIPTION
**Pull request by dhassaine**

### CODE REVIEW

File: src/containers/container.test.js [hunk 1]
```diff
@@ -2,12 +2,13 @@
...
 import PropTypes from "prop-types";
 import renderer from "react-test-renderer";
-import { Container, mapper, createStore } from "../main";
+import { Container, mapper, createStore } from "../react";
 
 class ErrorBoundary extends React.Component {
   componentDidCatch(error) {
     this.props.onError(error);
   }
+  e;
```
Comment (KrofDrakula) [inline src/containers/container.test.js]: Typo, I believe?

File: src/preact/index.test.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+/* global describe, it, expect */
+
+import { createStore } from "./index";
+
+describe("preact/index", () => {
+  it("smoke test", () => {
+    expect(createStore).toBeTruthy();
```
Comment (KrofDrakula) [inline src/preact/index.test.js]: You could asser it's a function. :)

File: src/containers/container.js [hunk 3]
```diff
@@ -1,81 +1,87 @@
...
+};
 
-export const mapper = (propMappings = {}, actionMappings = {}) => Wrapped => {
-  const PerfComponent = perf(Wrapped, Object.keys(propMappings));
-
-  return class Mapper extends React.Component {
-    static contextTypes = {
-      store: PropTypes.object
-    };
+/* @jsx h */
```
[the remark this answers is not in the document]
Comment (dhassaine) [reply src/containers/container.js]: Oh yeah, initially I thought you needed one of these comments per function that contains jsx.

File: src/containers/container.test.js (hunk 1 above)
Comment (dhassaine) [reply src/containers/container.test.js]: hmm, no idea how I did that!

**Comment by dhassaine** [comment]
@KrofDrakula  Thanks for taking a look and the comments. I've made the requested changes. In the Preact smoke test, I'm now asserting that all of the api is exported by comparing the exports to the keys generated by the vdom wrapper.

**Pull request merged**